Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:18-alpine as builder
+RUN apk --no-cache add bash
```

Comment:
why do you need to install bash?

Reply:
<img width="487" alt="image" src="https://github.com/saleor/saleor-dashboard/assets/40910668/a9aff21b-fba4-40f1-81d4-962be650e711">

Without bash this script cannot execute. And this script is executed while building the image.


![image](https://github.com/saleor/saleor-dashboard/assets/40910668/42303104-ec1d-476e-90a2-6b2f06ba54dd)

Reply:
basically, we don't use sentry at all within the docker but the script is being executed even if it skips the sentry-related scripts. Ok let's keep the bash for now, will perhaps wok on some workaround later.

Reply:
My goal was to make a docker image build. I didn't want to interfere with the script. I left the script changes for other issues.